__tests__/tests/components/CommentInput.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import {Platform} from 'react-native';
+
+import {CommentInput} from 'components';
+
+describe('<CommentInput />', () => {
+
+  const defaultProps = {
+    users: [],
+    userHasPushPermission: true,
+    issueLocked: false,
+    locale: '',
+    onSubmit: () => {},
+  };
+
+  it('should render TextInput and TouchableOpacity if I can post', () => {
```

Comment:
Use 3rd person. `'should render TextInput and TouchableOpacity if user has post permissions'`